beacon_node/execution_layer/src/engines.rs

```diff
@@ -159,7 +159,7 @@ impl Builder for Engine<BuilderApi> {
 /// Holds multiple execution engines and provides functionality for managing them in a fallback
```

Comment:
Perhaps we could quickly change this comment to say it holds a single execution engine?